lua/nvim-treesitter/health.lua

```diff
@@ -16,8 +16,8 @@ local NVIM_TREESITTER_MINIMUM_ABI = 13
 local function install_health()
   health.report_start "Installation"
 
-  if fn.has "nvim-0.7" == 0 then
-    health.report_error "Nvim-treesitter requires Neovim 0.7.0+"
+  if fn.has "nvim-0.8.0" ~= 1 then
+    health.report_error "Nvim-treesitter requires Neovim 0.8.0+"
```

Comment:
With vim.treesitter.is_in_node_range we even require 8.3 for some operations

Reply:
Latest stable is our policy

Reply:
It might be worth it to make this a runtime requirement, rather than a health check. Although, I'm not sure how that will play out with lazy-loading vs calling `setup()` manually.